__tests__/app.test.js

```diff
@@ -314,4 +314,134 @@ describe("POST /api/reviews/:review_id/comments", () => {
                 expect(body.msg).toBe("Not Found.")
             })
     })
+})
+
+describe("PATCH /api/reviews/:review_id", () => {
+    test('Responds with 200 and an updated review when decreasing votes.', () => {
+        const vote_updates = {
+            inc_votes: -10
+        };
+        return request(app)
+            .patch('/api/reviews/2')
+            .send(vote_updates)
+            .expect(200)
+            .then(( {body} ) => {
+                const {review} = body;
+                expect(review).toBeInstanceOf(Object);
+                expect(review).toEqual({
```

Comment:
Consider a more flexible jest matcher here, if our database is ever updated we don't want this test to fail and have to go back and change it

Reply:
I am sorry but I dont understand what you mean.